packages/atti-components/src/components/Slider/styles/SliderIndicator.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import Label from "../../Typographies/Label";
+import PropTypes from "prop-types";
+import SliderStep from "./SliderStep";
+
+const SliderIndicator = styled.span`
+  ${({leftPosition, moving, theme: {components: {slider: {sliderIndicator}}}}) => `
+    background-color: ${sliderIndicator.colors.backgroundColor};
+    border-radius: ${sliderIndicator.borders.radius};
+    height: ${sliderIndicator.sizes.height};
+    width: ${sliderIndicator.sizes.width};
+    left: ${`calc(${leftPosition}px - ${sliderIndicator.sizes.width.slice(0, -2) / 2}px)`};
+    
+    ${Label}{
+      display: ${moving ? 'block' : 'none'};
+      right: ${`calc(${sliderIndicator.sizes.width.slice(0, -2) / 2}px - ${sliderIndicator.valueLabel.width.slice(0, -2) / 2}px)`};
```

Comment:
Maybe add a comment to indicate that the `slice` is there to cut the `'px'` out of the values ?

Reply:
Done